CMakeLists.txt

```diff
@@ -303,15 +302,15 @@ add_rs_component_workspace(services/:Plugins
 
 set(XADMIN_PATH ${CMAKE_CURRENT_SOURCE_DIR}/services/user/XAdmin/ui)
 set(XADMIN_TRANSPILE_BYPRODUCT ${XADMIN_PATH}/wasm-transpiled/x_admin.core.wasm)
-add_js_target(${ADMIN_DIR} index.html ${XADMIN_TRANSPILE_BYPRODUCT} ${CommonApiCommonLib_js_DEP})
 add_dependencies(XAdmin_js XAdminWasm)
 
+# TODO: Remove this code; not sure how to yet
 add_custom_command(
     OUTPUT ${XADMIN_TRANSPILE_BYPRODUCT}
     DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/rust/release/psibase ${COMPONENT_BIN_DIR}/${XAdminWasm_OUTPUT_FILE}
     WORKING_DIRECTORY ${XADMIN_PATH}
-    COMMAND yarn --mutex network
-    COMMAND yarn jco transpile ${COMPONENT_BIN_DIR}/${XAdminWasm_OUTPUT_FILE} --minify --no-nodejs-compat --base64-cutoff=4096 --valid-lifting-optimization -o ${XADMIN_PATH}/wasm-transpiled
+    # COMMAND yarn
+    # COMMAND yarn jco transpile ${COMPONENT_BIN_DIR}/${XAdminWasm_OUTPUT_FILE} --minify --no-nodejs-compat --base64-cutoff=4096 --valid-lifting-optimization -o ${XADMIN_PATH}/wasm-transpiled
```

Comment:
james, this feels a little hacky, but I know I need to keep dependency and service list info in CMake for the sake of pacakges. So if this is right, pls just delete these 2 commented lines. If this needs to be/can be handled differently, please push the change to this branch as well